utils/include/detray/detectors/create_telescope_detector.hpp

```diff
@@ -353,6 +353,10 @@ auto create_telescope_detector(
     using detector_t = detector<telescope_types<typename mask_t::shape>,
                                 covfie::field, container_t>;
 
+    // Temporal restriction due to missing local navigation
+    assert((dists.size() - 1u < 20u) &&
```

Comment:
Hmm I would write it as `dists.size() <= 20u`

Reply:
You are right, this is wrong, but I was expecting that you need an extra slot for the exit portal, so it must be 
```
dists.size() < 20u
```

Reply:
mkay